src/libaktualizr/package_manager/packagemanagerfake.cc

```diff
@@ -65,7 +65,13 @@ data::InstallationResult PackageManagerFake::finalizeInstall(const Uptane::Targe
 
   data::InstallationResult install_res;
   if (target == targets[pending_version]) {
-    install_res = data::InstallationResult(data::ResultCode::Numeric::kOk, "Installing fake package was successful");
+    if (fiu_fail("fake_install_finalization_failure") != 0) {
+      install_res = data::InstallationResult(data::ResultCode::Numeric::kInstallFailed,
+                                             "Failed to finalize the pending update installation");
```

Comment:
What benefit does it bring in terms of installation failure ?

Reply:
None, really. I just wanted to be consistent with the other failure points and the docs that imply that that's how it should work, and it may be useful for QA and dev/testing of other parts of the system.

Reply:
Yes, agree that it can be useful for QA and dev testing.